src/font.rs

```diff
@@ -146,6 +145,7 @@ struct TableRecord {
 }
 /// The header of the font's table directory
 #[derive(Deserialize)]
+#[allow(non_snake_case)]
```

Comment:
I had this here before, and it's clearly the right thing, but my compiler - (1.54.0-nightly (5dc8789e3 2021-05-21)) - seemed to ignore it and warn about the non-snake case names anyway. That's why I made it module-level. Maybe I need to upgrade rustc...

Reply:
honestly I spent an hour banging my head against this, and I'm not 100% sure what's going on??